### DISCUSSION TITLE
Geometry

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3024] [Q&A]
I want to use the cylindrical geometry for my cell. Does it work with the existing pre defined parameter set?
Or how to use the cylindrical geometry in model?

**Comment by brosaplanella** [comment]
If you just want to model the electrochemistry, then the geometry does not make a difference. If you want to also model a temperature, I think for a cylindrical cell the only way is to use the lumped model and adjust the volume and surface area parameters for your cell.
